tests/behavioral/evalhub_adapter/tests/test_config_and_benchmarks.py

```
@@ -0,0 +1,294 @@
+        assert len(queries) == 2
+        assert isinstance(queries[0], QuerySpec)
+        assert queries[0].query == "What is 2+2?"
+        assert queries[0].expected_tools == ["calculator"]
+        assert queries[0].expected_elements == ["4"]
+        assert queries[1].query == "Search for cats"
+
+    def test_load_queries_missing_file(self, fixtures_dir: Path):
+        """A non-existent queries file raises FileNotFoundError."""
+        bm = BenchmarkDef(queries_file="does_not_exist.yaml", scorers=[])
+
+        with pytest.raises(FileNotFoundError, match="does_not_exist.yaml"):
```

Ruff (0.15.10)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)